commit 7db4c7826a538e0a3a59f1f5d6fca26de7895204
Author: Valentina Mukhamedzhanova <vmukhame@redhat.com>
Date:   Thu Jul 30 15:20:27 2015 +0200

    Fix fssnap_automatic_devices and fssnap_automatic_percentage in yum.conf manpage

diff --git a/docs/yum.conf.5 b/docs/yum.conf.5
index 22701e0..5993b57 100644
--- a/docs/yum.conf.5
+++ b/docs/yum.conf.5
@@ -894,12 +894,12 @@ How many old snapshots should yum keep when trying to automatically create a
 new snapshot. Setting to 0 disables this feature. Default is '1'.
 
 .IP
-\fBfssnap_automatic_percentage\fR
+\fBfssnap_percentage\fR
 The size of new snaphosts, expressed as a percentage of the old origin device. 
 Any number between 1 and 100. Default is '100'.
 
 .IP
-\fBfssnap_automatic_devices\fR
+\fBfssnap_devices\fR
 The origin LVM devices to use for snapshots. Wildcards and negation are allowed,
 first match (positive or negative) wins.
 Default is: !*/swap !*/lv_swap glob:/etc/yum/fssnap.d/*.conf
